src/ServiceControl.Config/Xaml/Behaviours/StylizedBehaviors.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace ServiceControl.Config.Xaml.Behaviours
 {
     using System.Windows;
-    using System.Windows.Interactivity;
+    using Microsoft.Xaml.Behaviors;
```

Comment:
This is used in ClearWindow class, which is used for our stylized dialogs. It can't be simply removed, as it will result in a look like this:

![image](https://user-images.githubusercontent.com/132117/139748385-d0a74a86-4a08-4be8-b0d4-ce0821909b8a.png)

Reply:
I was wondering if the framework had this as an option now but it seems not.